<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendWidget</class>
 <widget class="QWidget" name="FriendWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#labe_profile{
	background:url(:/Client/profile);
}

#label_content,#label_time{
	color:rgb(153, 153, 153)
}</string>
  </property>
  <widget class="QLabel" name="labe_profile">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>7</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>aðŸ¥³l</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_content">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>48</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>å¥½é¥±å‘€~</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12ï¼š00</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../res/Client.qrc"/>
 </resources>
 <connections/>
</ui>
